### ISSUE TITLE
6.0.0beta4 reset settings

### ISSUE THREAD

**Issue by CrYpT0n1T3** [jdf76/plugin.video.youtube#307]
I have installed beta 4 but the settings are resetting.
OSMC on raspberry pi

**Comment by anxdpanic** [comment]
I just made some additional changes to hopefully resolve this. Can you try ~beta5 and see if that resolves it for you, thank you.

**Comment by pannal** [comment]
I've got a similar problem with 6.0.0beta6 on current Krypton stable (OSMC, Vero4k).

Most of the time when rebooted MPEG DASH won't work (normal streams get used). Only after a bunch of trial and error with resetting and reinstalling the YouTube plugin, and restarting Kodi DASH will be enabled again.

Sometimes after reinstalling the addon, MPEG DASH is even greyed out and it needs a couple of reinstall/re-enable sessions until it's selectable again - also not guaranteeing it to work afterwards.

Should I open a separate issue?

**Comment by anxdpanic** [comment]
mpeg-dash and mpeg-dash proxy are required to be enabled. If you default your settings or mpeg-dash is greyed out, re-enabling dash requires opening the add-on once. (this will be changed as SPMC is no longer and doesn't support our dash anymore)

The mpeg-dash proxy should be run at startup, but will also be started if enabled and not running when opening the add-on. 

Will address the mentioned issue for beta7 today/tomorrow.

**Comment by pannal** [comment]
@anxdpanic awesome, thank you!

**Comment by anxdpanic** [comment]
@pannal I've posted beta7 if you'd like to give it a try.

**Comment by pannal** [comment]
Updated to beta 7, will report back. So far it's looking good.

One side question: before I've re-done my kodi profile I used beta 6 and always got the 1440p stream when watching youtube (I have a 1080p screen). Now it always selects the 1080p stream with 24p or 60p refresh rate, even though I've set it to ignore my screen size.

Is this intended behaviour? Does the fact that I use the screen refresh rate adaption setting of Kodi impact this perhaps?

**Comment by anxdpanic** [comment]
Not sure on refresh rate, may be your min bandwidth. If you set stream selection manual in inputstream.adaptive you can change video streams during playback by opening the video settings dialog(by subtitle menu on play screen). That way you can see if 1440 is available and working for you.

**Comment by pannal** [comment]
Thank you. I've just noticed this, should I care? `ERROR: GetDirectory - Error getting /home/osmc/.kodi/temp/plugin.video.youtube/`

**Comment by anxdpanic** [comment]
The temp folder  'error' is no harm, temp folder is removed at start up and created as needed.
For your particular video it may be that 1440p is a webm codec and not playable. I just searched 4K and tried a couple videos and am definitely seeing 1440p results.

**Comment by anxdpanic** [comment]
I'm going to close this issue as we're off topic, please open a new issue and include video id or log if resolution is still an issue.
